rel/config.exs

```diff
@@ -30,13 +30,13 @@ environment :dev do
   # dev mode.
   set dev_mode: true
   set include_erts: false
-  set cookie: :"mby8_WifcvH[I^0B0<)FSkUpadXF~hWXY`ct6=lY9kj5cotG*^KZAaz0u?RYY%%`"
+  set cookie: :"placeholder_cookie"
```

Comment:
I haven't tried, but since the cookie is set in vm_args anyway, couldn't this line be removed?

Reply:
Unfortunately not :/, distillery needs cookie here and erlang needs cookie in vm_args, thus I put here just some dummy token, since it's taken from vm_args.